corehq/apps/export/models/new.py

```diff
@@ -1341,6 +1342,8 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
         if inferred_schema:
             current_schema = cls._merge_schemas(current_schema, inferred_schema)
 
+        current_schema = cls._reorder_schema_from_app(current_schema, app_id, identifier)
```

Comment:
we could probably be smarter about saving the fetched app/schema during when we process the apps, but this logic is cleaner and simpler so i opted for it. now since we only process the most recent app, generating an export isn't as slow as it used to be